documentation/src/main/markdown/downloadingandrunning/ephemeralscan.md

```diff
@@ -0,0 +1,112 @@
+# Ephemeral Scan
+
+Ephemeral Scan or Ephemeral Scan Mode is a new way of running [solution_name] with Black Duck. This mode is designed to be as fast as possible and does not persist any data on Black Duck. Ephemeral Scan Mode has a unique set of restrictions, mode of configuration and set of results.  It is similar to Rapid Scan Mode however it differs in that it supports usage of the SIGNATURE_SCAN tool.  Ephemeral Scan allows non-persistent signature scans to be performed.
+
+Enable this feature by adding [--detect.blackduck.scan.mode=EPHEMERAL](../properties/configuration/blackduck-server.md#detect-scan-mode-advanced) to a run of detect.
+
+## Requirements and Limitations
+
+ * A limited subset of Tools can be run. 
+    * The currently supported tools are: DETECTOR, BAZEL, SIGNATURE_SCAN and DOCKER.
+    * The EPHEMERAL Signature Scan will not persist on black duck. 
+    * All other tools are disabled when running in Ephemeral Scan mode.
+ * Ephemeral Scan and non-persistent SIGNATURE_SCAN
+    * To perform a non-persistent Signature Scan in Ephemeral mode, SIGNATURE_SCAN must be included within --detect.tools.
+    * Permitted tools omitted from the detect.tools list will not be run.
+ * Ephemeral Scan requires Black Duck policies. 
+    * Ephemeral Scan only reports components that violate policies. 
+    * If no policies are violated or there are no defined policies, then no components are returned.
+ * Ephemeral Scan does not support ```detect.policy.check.fail.on.severities```
+    * [solution_name] will fail with FAILURE_POLICY_VIOLATION if any component violates Black Duck polices with a CRITICAL or BLOCKER severity. 
+    * See the Black Duck documentation for a list of policy conditions that are supported by Ephemeral Scan. 
+ * Ephemeral Scan does not support ```detect.policy.check.fail.on.names```
+ * Ephemeral Scan cannot create a Risk or Notices report.
+ * Ephemeral Scan will not create a Project or Version on Black Duck.
+ * Ephemeral Scan should be run with aggregation so server-side metrics can be accurately collected.
```

Comment:
Perhaps add the link to the BDIO aggregation page here?

Reply:
good idea